skbeam/core/utils.py

```diff
@@ -1215,6 +1217,51 @@ def geometric_series(common_ratio, number_of_images, first_term=1):
 
     geometric_series = [first_term]
 
-    while geometric_series[-1]*common_ratio < number_of_images:
-        geometric_series.append(geometric_series[-1]*common_ratio)
+    while geometric_series[-1] * common_ratio < number_of_images:
+        geometric_series.append(geometric_series[-1] * common_ratio)
     return geometric_series
+
+
+def bin_grid(image, r_array, pixel_sizes, statistic='mean', mask=None,
```

Comment:
Sorry @CJ-Wright  I just caught this. Why is `pixel_sizes` a mandatory parameter? If `bins` is specified, it shouldn't be necessary. Perhaps you should make a change? I'm also fine to do it if you're okay with it.

Fortunately, this can be changed to 
```python
bin_grid(image, r_array, pixel_sizes=None, statistic='mean'
```
without breaking the API.